### PR TITLE
Moz context properties

### PR DESCRIPTION
**Pull request by chrisdavidmills**
Making changes according to jwatt's comments on the other related pull request

### CODE REVIEW

**Comment by Elchi3** [comment]
> One comment — I've set "appliesto" as "blockLevelElements", as I'm not 100% sure what to set this as. I don't think it's exactly right, but it is closer than "replacedElements". ;-) I'll have to research what this needs to be changed to.

Can we try to answer this question? `appliesTo` has a lot of enums:
https://github.com/mdn/data/blob/master/css/properties.schema.json#L153

There is `allElements` or `images`, for example.

**Comment by chrisdavidmills** [comment]
I've had a look, and don't think any of these particularly suit the situation perfectly.

Can we just add one? What is the process here? I'd put something like 

allElementsThatCanReferenceImages

**Comment by Elchi3** [comment]
I believe the process would be:

1) Add `"appliesto": "allElementsThatCanReferenceImages" ` for `-moz-context-properties` to `css/properties.json`.
2) Add `"allElementsThatCanReferenceImages" ` to the enum list in `css/properties.schema.json#L153`
3) Add `"allElementsThatCanReferenceImages" ` as a new l10n object to `l10n/css.json` and describe what it means there (this sentence will appear rendered on MDN in the end)

File: l10n/css.json [hunk 1]
```diff
@@ -345,6 +345,9 @@
     "ja": "全要素. In SVG, it applies to container elements, graphics elements, and graphics referencing elements.",
     "ru": "Все элементы. В SVG это применяется к контейнерам, графическим элементам и элементам графической отсылки."
   },
+  "allElementsThatCanReferenceImages": {
+    "en-US": "Any element that can have an image applied to it, for example as a {{cssxref(\"background-image\")}}, {{cssxref(\"border-image\")}}, or {{cssxref(\"list-style-image\")}}.",
```
Comment (lahmatiy) [inline l10n/css.json]: Bad comma at the ending (the reason checking failed)

Comment (chrisdavidmills) [inline l10n/css.json]: D'oh! Thanks lahmatiy - comma removed.

**Pull request merged**